openedx/core/djangoapps/content_staging/api.py

```diff
@@ -1,4 +1,100 @@
 """
 Public python API for content staging
 """
-# Currently, there is no public API.
+from __future__ import annotations
+from datetime import datetime
+from typing import NamedTuple
+
+from django.http import HttpRequest
+from opaque_keys.edx.keys import UsageKey
+
+from .models import UserClipboard as _UserClipboard, StagedContent as _StagedContent
+from .serializers import UserClipboardSerializer as _UserClipboardSerializer
```

Comment:
Interesting way to keep people from importing these from the wrong place. I'm not sure if I prefer it to importing the module and doing `models.UserClipboard`, but I'm fine with it, fwiw.

Reply:
I actually thought this is what OEP-49 recommends? "When importing internal app code to be used in the api.py file, prefix it with an underscore so it’s clear it’s for internal use only." Or are you saying the convention is to import `models as _models` and use that?

Reply:
You're right, it's there. I must have just forgotten about that part, because I haven't been writing my code like that. 😛 I'll try to do so going forward.

I actually meant:

```python
from . import models
```

So that all references are to `models. UserClipboard`. Granted, it doesn't mean people can't re-import it, but it's less likely.